docker_images/ci-functional/Dockerfile

```diff
@@ -85,6 +85,13 @@ RUN curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-instal
     && pyenv install $PY36 \
     && pyenv install $PY27
 
+# Install Ninja
+
+RUN wget https://github.com/ninja-build/ninja/archive/v1.10.2.tar.gz \
```

Comment:
Not sure why it is passing develop2 now: https://ci.conan.io/blue/organizations/jenkins/ConanTestSuitev2/detail/develop2/91/pipeline/262

I added this, because ci-functional didn't seem to have ninja installed, and it was failing.

Reply:
Oh, nevermind, it is because develop2 still doesn't use ci_functional docker image, it uses the older ones.